qiling/profiles/uefi.ql

```diff
@@ -24,15 +22,14 @@ heap_size		= 0x02000000
 [HOB_LIST]
 # EFI_GLOBAL_VARIABLE
 Guid = 7739f24c-93d7-11d4-9a3a-0090273fc14d
-Table = 0x00000000ffffffff
+# the HOB list must end with an etry type of FFFF
```

Comment:
etry? do you mean entry? Maybe `HobType of FFFF`

Reply:
Indeed; this is a typo. Should have been "_The HOB list must end with an entry whose HobType is FFFF_"